tasks.js

```diff
@@ -112,4 +112,38 @@ module.exports = [
       },
     }],
   },
+
+  // Symptomatic contact follow up
+  {
+    name: 'symptomatic_contact_follow_up',
+    icon: 'icon-healthcare',
+    title: 'task.symptomatic_contact_follow_up.title',
+    appliesTo: 'contacts',
+    appliesToType: undefined,
+    appliesIf: function (contact) {
+      this.mostRecentQuarantineFollowUp = Utils.getMostRecentReport(contact.reports, 'QUARANTINE_FOLLOW_UP');
```

Comment:
The form to look at here is the `symptoms_check` form instead not the quarantine_follow_up

Reply:
Updated